lib/widgets/navigation_drawer.dart

```diff
@@ -93,7 +93,54 @@ Widget _breezDrawerHeader(UserProfileBloc user, bool drawAvatar) {
   return new BreezDrawerHeader(
     padding: EdgeInsets.only(left: 16.0),
     child: !drawAvatar
-        ? new Container()
+        ? new StreamBuilder<BreezUserModel>(
+            stream: user.userStream,
+            builder: (context, snapshot) {
+              if (!snapshot.hasData) {
+                return new Container();
+              } else {
+                return new Column(children: <Widget>[
```

Comment:
I believe this is a duplicate from the user app, let's extract out into a widget.

Reply:
[Done](https://github.com/breez/breezmobile/pull/205/commits/bd3b25fa17eb5766f00fb9b756f0ea0e7409deaa)